Sources/TensorFlow/Operators/NN.swift

```diff
@@ -409,15 +408,14 @@ public func conv2D<Scalar: TensorFlowFloatingPoint>(
         explicitPaddings: [])
 }
 
-/// Computes a 3-D convolution using `self` as input, with the specified
-/// filter, strides, and padding.
+/// Computes a 3-D convolution with the specified input, filter, strides, and padding.
 ///
 /// - Parameters:
 ///   - x: The input.
 ///   - filter: The convolution filter.
 ///   - strides: The strides of the sliding filter for each dimension of the input.
 ///   - padding: The padding for the operation.
-/// - Precondition: `self` must have rank 5.
+/// - Precondition: `input` must have rank `5`.
```

Comment:
Same issue here.

Reply:
@rxwei why did we change it to input in #241 then o.O? I'll change it to x there as well. cool.

Reply:
I actually suggested changing all the `x`'s to `input`.

Reply:
@rxwei what about the comments in the parameters?

Reply:
Yes.

Reply:
@rxwei one final thing, Should I change all the x's in nn.swift to input?

Reply:
For convolution and pooling layers, yes.